src/util/resolve-dir.js

```diff
@@ -14,9 +14,12 @@ export default async (dir) => {
     if (await fs.exists(testPath)) {
       const packageJSON = JSON.parse(await fs.readFile(testPath, 'utf8'));
 
-      if (packageJSON.devDependencies && packageJSON.devDependencies['electron-prebuilt-compile']
-          && !/[0-9]/.test(packageJSON.devDependencies['electron-prebuilt-compile'][0])) {
-        global._resolveError = () => console.error('You must depend on an EXACT version of "electron-prebuilt-compile" not a range'.red);
+      if (packageJSON.devDependencies && packageJSON.devDependencies['electron-prebuilt-compile']) {
+        if (!/[0-9]/.test(packageJSON.devDependencies['electron-prebuilt-compile'][0])) {
+          global._resolveError = () => console.error('You must depend on an EXACT version of "electron-prebuilt-compile" not a range'.red);
+        }
+      } else {
```

Comment:
I thought I had flaky tests already and was about to cry 😆 Quite happy that it caught it 👍